#DualShock4 State message
DualShock4ButtonState btn_state
DualShock4HatState hat_state
DualShock4JoyStickState joy_state
